security/README.md

```diff
@@ -0,0 +1,42 @@
+Security
+========
+
+A guide for practicing safe web.
+
+Think
+-----
+
+Security is important, and you can't practice these guidelines without
+understanding them. Question each guideline before following it yourself and
+start a discussion whenever you're in doubt.
+
+Passwords
+---------
+
+* Use a unique password for every account you create.
+* Don't store passwords in plain text.
+
+HTTPS
+-----
+
+* Use HTTPS for all web traffic.
+* Use HTTPS in the beginning; it's harder to introduce later.
+* Use HTTPS redirects for HTTP traffic.
+* Use HSTS headers to enforce HTTPS traffic.
```

Comment:
I think this will be new for a lot of people, so a link would seem useful. Maybe http://tools.ietf.org/html/rfc6797